packages/i18n/src/ja.ts

```diff
@@ -0,0 +1,66 @@
+import { Language } from './types';
+
+// prettier-ignore
+const language: Language = {
+  code:             'ja',
+  schema:           (issue) => `無効な型: ${issue.expected} を期待したが、 ${issue.received} を受け取りました`,
```

Comment:
I forgot to ask for a correction.
As @t6adev suggests, it would be better to change `期待したが` to `期待しましたが` to make the whole message feel more unified.